src/tests/components/common/userProfile.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { findTestByAttr, store } from '../../../../helpers/utils/testUtils';
+import { UserProfile } from '../../../components/common/userProfile';
+
+const props = {
+  getNotifs: jest.fn(),
+  auth: { user: {} }
+};
+const state = {
+  profile: '',
+  notifs: []
+};
+const nextProps = {
+  userInfo: {
+    notifFetched: true,
+    notifs: []
+  }
+};
+let buttons = null;
+const userProfile = mount(
+  <Provider store={store}>
+    <MemoryRouter>
+      <UserProfile {...props} />
+    </MemoryRouter>
+  </Provider>
+);
+describe('<viewProfile/>', () => {
+  beforeEach(() => {
+    buttons = userProfile.find('img');
+
+    buttons.map(btn => btn.simulate('click', {}));
+    expect(userProfile).toHaveLength(1);
```

Comment:
why are expecting in a before each hook, this is not the standard practise please refactor this